webrtc.html

```diff
@@ -8713,6 +8713,15 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           values in <var>configuration</var>.</p>
         </li>
         <li>
+          <p>Initialize <var>channel</var>'s
+          <a data-for="RTCDataChannel"><code>priority</code></a> attribute to
+          the <a>RTCDataChannelPriority</a> enum value that corresponds to the
+          integer priority value in <var>configuration</var>, as defined in
```

Comment:
The value in *configuration* is an enum though, not an integer. It's an enum that gets turned into an integer, for the "underlying data transport", not the other way around.